staking/types/transaction.go

```diff
@@ -39,7 +39,11 @@ func (d *txdata) CopyFrom(d2 *txdata) {
 	restored, _ := RLPDecodeStakeMsg(
```

Comment:
Should we also directly change d2.StakeMsg == nil? and avoid RLPDecodeStakeMsg?

Reply:
but in this case, we don't know if RLPDecodeStakeMsg will success or not, only if it returns nil